server/db/migration/migrations/sqls/20211111111648-create-users-tables-up.sql

```diff
@@ -0,0 +1,44 @@
+create type users_status as enum ('invitationPending', 'active', 'inactive');
+
+create table users
+(
+    institution              varchar(1024),
+    lang                     varchar(2) default 'en'::character varying not null,
+    id                       bigserial
+        constraint users_pkey
+            primary key,
+    profile_picture_filename varchar(1024),
+    name                     varchar(1024),
+    status                   users_status default 'invitationPending' not null,
+    profile_picture_file     bytea,
+    position                 varchar(1024),
+    email                    varchar(255) not null
+);
+
+create type provider as enum ('local', 'google');
+
+create table users_provider
+(
+    id                      bigserial
+        constraint users_provider_pkey
+            primary key,
+    user_id                 bigint references users (id) not null,
+    provider                provider not null,
+    props                   jsonb
+);
+
+create table users_invitation
+(
+    uuid                    uuid primary key,
+    invited_at              timestamptz,
+    accepted_at             timestamptz,
+    user_id                 bigint references users (id) not null
+);
+
+create table users_reset_password
+(
+    uuid                    uuid primary key,
+    changed_at              timestamptz,
+    created_at              timestamptz,
+    user_id                 bigint references users (id) not null
+);

```

Comment:
for timestamptz isn't better to use what we use in other migrations as well?

Reply:
Let's use timestamptz from now on

```
The big difference between these two data types is that timestamptz includes a timezone offset while timestamp does not. So it is important to decide for the timestamp you are storing whether the timezone is important to keep or not. 
```
https://kb.objectrocket.com/postgresql/postgresql-timestamp-vs-timestamptz-616